receiver/awsxrayreceiver/internal/translator/cause.go

```diff
@@ -97,14 +97,19 @@ func convertStackFramesToStackTraceStr(excp awsxray.Exception) string {
 	b.WriteString(*excp.Message)
 	b.WriteString("\n")
 	for _, frame := range excp.Stack {
-		line := strconv.Itoa(*frame.Line)
+		label := awsxray.StringOrEmpty(frame.Label)
+		path := awsxray.StringOrEmpty(frame.Path)
+		line := ""
```

Comment:
```suggestion
		line := "<unknown>"
```

Maybe have a placeholder line number to indicate that it was not provided?

Reply:
Yep. Sure.